spec/controllers/admin/entries_controller_spec.rb

```diff
@@ -455,20 +458,21 @@ def self.name
   end
 
   describe '#duplicate' do
-    it 'does not allow account manager to duplicate entries of other account' do
-      account = create(:account)
-      entry = create(:entry)
+    it 'does not allow account editor to duplicate entries of own account' do
+      user = create(:user)
+      account = create(:account, with_editor: user)
+      entry = create(:entry, account: account)
 
-      sign_in(create(:user, :account_manager))
+      sign_in(user)
 
       expect {
         post(:duplicate, :id => entry.id)
       }.not_to change { Pageflow::Entry.count }
     end
 
-    it 'allows account manager to duplicate entries of own account' do
-      user = create(:user, :account_manager)
-      entry = create(:entry, :account => user.account)
+    it 'allows entry publisher to duplicate entry' do
+      user = create(:user)
+      entry = create(:entry, with_publisher: user)
```

Comment:
Let's rename these factory attributes to `with_publisher_member`, `with_editor_member` etc. Especially `with_publisher` sounds a lot like the user actually already published the entry. What do you think?